src/pixelmapFeature.js

```diff
@@ -0,0 +1,451 @@
+ *   transformations for those two triangles.
+ * @returns {geo.pixelmapFeature}
+ */
+//////////////////////////////////////////////////////////////////////////////
+
+//////////////////////////////////////////////////////////////////////////////
+var pixelmapFeature = function (arg) {
+  'use strict';
+  if (!(this instanceof pixelmapFeature)) {
+    return new pixelmapFeature(arg);
+  }
+  arg = arg || {};
+  feature.call(this, arg);
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * @private
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  var m_this = this,
+      m_quadFeature,
+      m_srcImage,
+      m_info,
+      s_update = this._update,
+      s_init = this._init,
+      s_exit = this._exit;
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Get/Set position accessor
+   *
+   * @returns {geo.pixelmap}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.position = function (val) {
+    if (val === undefined) {
+      return m_this.style('position');
+    } else if (val !== m_this.style('position')) {
+      m_this.style('position', val);
+      m_this.dataTime().modified();
+      m_this.modified();
+    }
+    return m_this;
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Get/Set url accessor
+   *
+   * @returns {geo.pixelmap}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.url = function (val) {
+    if (val === undefined) {
+      return m_this.style('url');
+    } else if (val !== m_this.style('url')) {
+      m_srcImage = m_info = undefined;
+      m_this.style('url', val);
+      m_this.dataTime().modified();
+      m_this.modified();
+    }
+    return m_this;
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Get the maximum index value from the pixelmap.  This is a value present in
+   * the pixelmap.
+   *
+   * @returns {geo.pixelmap}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.maxIndex = function () {
+    if (m_info) {
+      /* This isn't just m_info.mappedColors.length - 1, since there
+       * may be more data than actual indices. */
+      if (m_info.maxIndex === undefined) {
+        m_info.maxIndex = 0;
+        for (var idx in m_info.mappedColors) {
+          if (m_info.mappedColors.hasOwnProperty(idx)) {
+            m_info.maxIndex = Math.max(m_info.maxIndex, idx);
+          }
+        }
+      }
+      return m_info.maxIndex;
+    }
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Get/Set mapColor accessor
+   *
+   * @returns {geo.pixelmap}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.mapColor = function (val) {
```

Comment:
@manthey I am wondering why don't we just call it color like other features? since it is primarily a color transfer function? Also, I would think this should be a style element.